<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="5010" started-at="2021-03-06T07:04:19Z" finished-at="2021-03-06T07:04:24Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="5010" started-at="2021-03-06T07:04:19Z" finished-at="2021-03-06T07:04:24Z">
      <class name="demo.ExtentReportDemo">
        <test-method status="PASS" signature="loginTest()[pri:0, instance:demo.ExtentReportDemo@448ff1a8]" name="loginTest" duration-ms="316" started-at="2021-03-06T01:04:23Z" finished-at="2021-03-06T01:04:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
      </class> <!-- demo.ExtentReportDemo -->
      <class name="demo.ExtentReportDemo2">
        <test-method status="PASS" signature="setup()[pri:0, instance:demo.ExtentReportDemo2@51b7e5df]" name="setup" is-config="true" duration-ms="104" started-at="2021-03-06T01:04:19Z" finished-at="2021-03-06T01:04:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginTest()[pri:0, instance:demo.ExtentReportDemo2@51b7e5df]" name="loginTest" duration-ms="3510" started-at="2021-03-06T01:04:20Z" finished-at="2021-03-06T01:04:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at org.testng.Assert.assertTrue(Assert.java:47)
	at demo.ExtentReportDemo2.loginTest(ExtentReportDemo2.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:demo.ExtentReportDemo2@51b7e5df]" name="tearDown" is-config="true" duration-ms="323" started-at="2021-03-06T01:04:23Z" finished-at="2021-03-06T01:04:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult: loginTest STATUS:FAILURE METHOD:ExtentReportDemo2.loginTest()[pri:0, instance:demo.ExtentReportDemo2@51b7e5df]]
]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:demo.ExtentReportDemo2@51b7e5df]" name="endReport" is-config="true" duration-ms="740" started-at="2021-03-06T01:04:24Z" finished-at="2021-03-06T01:04:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- demo.ExtentReportDemo2 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
